integration_tests/tests/v2Manage/future_manager/createsOutOfServiceBed.cy.ts

```diff
@@ -85,25 +84,33 @@ context('OutOfServiceBeds', () => {
     page.shouldShowErrorMessagesForFields(['startDate', 'endDate', 'reason', 'notes'])
   })
 
-  it('should show an error when there are booking conflicts', () => {
+  it('should show an error when there are out of service bed conflicts', () => {
     // Given I am signed in with permission to create an out of service bed
-    signIn([], ['cas1_out_of_service_bed_create'])
+    signIn([], ['cas1_out_of_service_bed_create', 'cas1_view_out_of_service_beds'])
 
+    const bed = { name: 'abc', id: '123' }
     const premises = extendedPremisesSummaryFactory.build()
-    const conflictingBooking = bookingFactory.build()
+    const conflictingOutOfServiceBed = outOfServiceBedFactory.build({
+      bed,
+      startDate: '2022-02-11',
+      endDate: '2022-03-11',
```

Comment:
I don't think it matters that the OOSB built here is defined -- it could be a blank factory build. All we care about is that the response returns a conflict error, I think?